gammapy/spectrum/tests/test_spectrum_fit.py

```diff
@@ -10,6 +10,10 @@
 from ...utils.testing import requires_dependency, requires_data, SHERPA_LT_4_8
 from astropy.utils.compat import NUMPY_LT_1_9
 
+
+#TODO : PHACount should not expect a BIN_LO and BIN_HI columns
+#https://travis-ci.org/gammapy/gammapy/jobs/136170992#L939 
```

Comment:
No thats's fine. OK to merge. I will fix the test that had to be marked as xfail.